dispatch-client/src/main/scala/org/htt4s/client/dispatch/DispatchClient.scala

```diff
@@ -0,0 +1,50 @@
+package org.htt4s.client.dispatch
+
+import dispatch._, Defaults._
+import com.ning.http.client.{Response => DResponse}
+
+import org.http4s._
+import org.http4s.client._
+
+import scala.collection.JavaConverters._
+import scalaz.concurrent.Task
+
+import scalaz._
+import Scalaz._
+
+object DispatchClient {
+  def apply(): Client = {
+    Client(Service.lift { req =>
+      Task.async[DResponse] { register =>
+        Http(toDRequest(req)).onComplete {
+          case scala.util.Success(res) => register(res.right)
+          case scala.util.Failure(ex) => register(ex.left)
+        }
+      }.flatMap(fromDResponse)
+    }, Task.now(()))
```

Comment:
I admit to being a bit ignorant on async-http-client.  I'm surprised that there isn't a close method on the underlying response, to indicate that the client has no intention to read the body and that the connection can be freed.  (This is the case that motivated the `dispose` task in the first place.)  I feel like we're both missing something here.